playbooks/archivematica-centos7/singlenode.yml

```diff
@@ -0,0 +1,69 @@
+---
+- hosts: "am-local-centos7"
+
+  pre_tasks:
+
+    - include_vars: "vars-singlenode.yml"
+      tags:
+        - "always"
+
+    - name: "Set SELinux to Permissive"
+      command: "setenforce Permissive"      
+      become: "yes"
+
+    - name: "Enable epel repository"
+      yum:
+        name: "epel-release"
+        state: "latest"
+      become: "yes"
+
+    - name: "Install mariadb-server, gearman"
+      yum:
+        name: "{{ item }}"
+        state: "latest"
+      with_items:
+        - "mariadb-server"
+        - "gearmand"
+      become: "yes"
+
+    - name: "Enable and start mariadb, gearman servers"   
+      service:
+        name: "{{ item }}"
+        state: "started"
+        enabled: "yes"
+      with_items:
+        - "mariadb"
+        - "gearmand"
+      become: "yes"
+
+  roles:
+    - { role: "elastic.elasticsearch", 
```

Comment:
AFAIK that's just the syntax for inline mappings, see http://yaml.org/spec/1.0/#id2570829.